solutions/c/02-cz2/diff/app/main.c.diff

```diff
@@ -0,0 +1,16 @@
+@@ -1,11 +1,15 @@
+ #include <stdio.h>
++#include <string.h>
+
+ int main() {
+   printf("$ ");
+   fflush(stdout);
+
+   // Wait for user input
+   char input[100];
+   fgets(input, 100, stdin);
++  input[strlen(input) - 1] = '\0';
```

Comment:
@andy1li it might not be clear what this doing - this is removing the ending newline yes? Would recommend adding a comment for this if so